app/scripts/lib/constants.js

```diff
@@ -24,6 +24,9 @@ define([], function () {
 
     OAUTH_CODE_LENGTH: 64,
 
+    DERIVED_KEY_LENGTH: 32,
+    DERIVED_KEY_INFO_PREFIX: 'identity.mozilla.com/picl/v1/oauth/',
```

Comment:
> Any other particular reason for putting them here?

That was my request, more from project convention than from any hard rule. Perhaps constants.js has outlived its usefulness.